Healthcare-Hospital-Management-System/Healthcare-Hospital-Management-System/Program.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using HealthcareHospitalManagementSystem.Models;
+using HealthcareHospitalManagementSystem.Services;
+
+namespace HealthcareHospitalManagementSystem
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```

Comment:
Instead of using WebAPI you are using Console App. 
It is a requirement for the assignment to make it WebAPI project.